src/generator.py

```diff
@@ -52,7 +53,7 @@ def get_c_size(self, t):
         # this regex will get the X in u/intX_t (uint8_t, int16_t)
         match = re.search('[0-9]{1,2}', t)
         if match:
-            return int(match.group(0)) / 8
+            return int(match.group(0) / 8)
```

Comment:
Does it work?
I expected the result of  `match.group()` to be a string, which should not be divisible by 8.

Reply:
it does with python3 :) and travis ci

Reply:
> I expected the result of match.group() to be a string, which should not be divisible by 8.

though, that is probably why I found it the way it was. We (I) started development of this repo with python 2.